tests/unit/config/config_test.py

```diff
@@ -2121,7 +2121,7 @@ def test_resolve_environment_from_env_file_with_empty_values(self):
                 'FILE_DEF': u'bär',
                 'FILE_DEF_EMPTY': '',
                 'ENV_DEF': 'E3',
-                'NO_DEF': None
+                'NO_DEF': ''
```

Comment:
This change is wrong, the `None` is the correct value here, so I think the code change isn't correct to fix the issue.